ftw/keywordwidget/configure.zcml

```diff
@@ -83,4 +83,12 @@
         directory="./upgrades/select2js"
         />
 
+    <browser:page
+        for="z3c.form.interfaces.ISelectWidget"
```

Comment:
Why for all z3cform select widget? Shouldn't it be `ftw.keywordwidget` specicifc?